### PR TITLE
improve the log when encountering invalid checkpoint

### PR DESCRIPTION
**Pull request by kennytm**
### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update the `tidb-ansible` repository
 - Need to be included in the release note

### CODE REVIEW

**Comment by kennytm** [comment]
PTAL @leoppro @GregoryIan 

(CI is broken, to be fixed by SRE.)

File: lightning/restore/restore.go [hunk 1]
```diff
@@ -592,7 +594,54 @@ func (rc *RestoreController) restoreTables(ctx context.Context) error {
...
+			action.WriteString("./tidb-lightning-ctl --checkpoint-errors-")
+			switch failedStep {
+			case CheckpointStatusAlteredAutoInc, CheckpointStatusAnalyzed:
+				action.WriteString("ignore")
+			default:
+				action.WriteString("destroy")
+			}
+			action.WriteString("='")
+			action.WriteString(tableName)
+			action.WriteString("' --config=...")
```
Comment (lance6716) [inline lightning/restore/restore.go]: Didn't find `--config` in doc or cmd/tidb-lightning-ctl/main, what is this used for?

**Comment by kennytm** [comment]
Ping @leoppro @GregoryIan

File: lightning/config/global.go [hunk 2]
```diff
@@ -115,7 +115,7 @@ func LoadGlobalConfig(args []string, extraFlags func(*flag.FlagSet)) (*GlobalCon
 	pdAddr := fs.String("pd-urls", "", "PD endpoint address")
 	dataSrcPath := fs.String("d", "", "Directory of the dump to import")
 	importerAddr := fs.String("importer", "", "address (host:port) to connect to tikv-importer")
-	backend := fs.String("backend", "", `delivery backend ("importer" or "mysql")`)
+	backend := fs.String("backend", "", `delivery backend ("importer" or "tidb")`)
```
Comment (leoppro) [inline lightning/config/global.go]: where we handle the `backend` config? Should we changed the logic about `backend` config?

Comment (kennytm) [reply lightning/config/global.go]: This is to fix a typo. The setting should be `tidb` instead of `mysql`.

**Pull request merged**